docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE.md

```diff
@@ -175,7 +175,7 @@ distribution_desc
     
     `ENGINE=olap`
     
-* `key_desc`
+* `keys_type`
```

Comment:
the English docs has the same problem, please fix them together.

Reply:
The problem has been fixed both in EN and ZH docs